### DISCUSSION TITLE
CMU 17-803: Empirical Methods - CS自学指南

### DISCUSSION THREAD

**Discussion by giscus** [PKUFlyingPig/cs-self-learning#616] [Announcements]
# CMU 17-803: Empirical Methods - CS自学指南

CS自学指南

https://csdiy.wiki/%E8%BD%AF%E4%BB%B6%E5%B7%A5%E7%A8%8B/17803/

**Comment by RNCHEN** [comment]
！居然能看到我上过的课哈哈哈哈

**Reply by RNCHEN** [reply]
我以为是一门非常小众的课 :)

**Comment by ZhangLe5201314** [comment]
可以参考研究哈

**Comment by yigudove** [comment]
不是pku的，但是还是对哥们的无私分享致以敬意

**Comment by pnyyb** [comment]
请问这给网站的内容真的不会太难吗，适合我这个逻辑能力一般和数学差的看吗

**Comment by Teneeduu** [comment]
GREAT MAN

**Comment by Solomonpark** [comment]
好厉害

**Comment by zerorobot0** [comment]
请作者喝杯下午茶以表示敬意！

**Reply by zerorobot0** [reply]
致敬每一个有开源精神的人❤️

**Comment by Weakstupidchatbox** [comment]
麻烦问下有计算理论相关的好课吗

**Comment by ShangGuan2613** [comment]
请您喝杯咖啡哈哈哈

**Comment by KexinotKevin** [comment]
081203来自救了

**Comment by Dinversky** [comment]
现在看cs61a的课程点什么链接都会跳转到伯克利学校账号登录的页面,请问这个问题怎么解决

**Reply by MTDickens** [reply]
不是有 YouTube 链接吗？不要点 recordings，点 videos。

**Comment by cyansoft2017** [comment]
谢谢

**Comment by tzango** [comment]
双非cs小菜鸡向你表示感谢

**Comment by Haru-LCY** [comment]
Thanks♪(･ω･)ﾉ

**Comment by Su1ren** [comment]
伟大，无需多言

**Comment by Yuan-1028** [comment]
话说大佬们，可以有HCI相关的课吗？想自学一下关于HCI的东西😊

**Comment by lanwm0** [comment]
准大一，不是很自律，想组队一起学习，有意者加q2632952456（<● ●>)

**Comment by utsusemioxo** [comment]
thank you! This website has helped a lot people!!! AWESOME!!!

**Comment by ZngYM** [comment]
有没有XJTU的 求自学伙伴

**Comment by lyysxb** [comment]
作者是真的顶，可惜我都毕业了
